src/ansys/mapdl/core/_version.py

```diff
@@ -8,7 +8,7 @@
 """
 
 # major, minor, patch
-version_info = 0, 60, "dev0"
+version_info = 0, 61, "dev0"
```

Comment:
You might consider fetching this from the package info using importlib_metadata

Reply:
Agreed. @germa89, can you implement that? See:
https://github.com/pyansys/template/blob/6c53fe872f57d346f5555f2d8025c8e49b432064/src/ansys/product/library/__init__.py#L24-L29

Reply:
Done.

```py
try:
    import importlib.metadata as importlib_metadata
except ModuleNotFoundError:
    import importlib_metadata

# Read from the pyproject.toml
# major, minor, patch
__version__ = importlib_metadata.version('ansys-mapdl-core')
```

This does not work if you change `pyproject.toml` version locally and restart python. It seems it is directly linked to the version installed. I wonder who this will behave after we do a release.

Reply:
> It seems it is directly linked to the version installed.

That's correct as the metadata is modified when installing with `pip`.

Single source versioning is going to have drawbacks no matter what we choose to implement I think. Developers will be smart enough to work around this I think, but we can always revert if necessary.

Reply:
I felt we should have relied in directly reading the `pyproject.toml` file and grabbing the version directly from there. But I didn't see any good method to reliable open that file. And everybody seems to be using this for some reason.